### ISSUE TITLE
The input of the forward part of my model is a tuple, which cannot be converted to onnx format according to the existing methods. Can you tell me how to solve it

### ISSUE THREAD

**Issue by singaln** [pytorch/pytorch#77341]
### 🐛 Describe the bug

import torch
import torch.nn as nn

class Model(nn.Module):
    def __init__(self):
        super(Model, self).__init__()
        self.conv1 = nn.Linear(32, 16)
        self.relu1 = nn.ReLU(inplace=True)
        self.relu2 = nn.ReLU(inplace=True)
        self.fc = nn.Linear(32, 2)

    def forward(self, x):
        x1, x2 = x
        x1 = self.conv1(x1)
        x1 = self.relu1(x1)
        x2 = self.conv1(x2)
        x2 = self.relu1(x2)
        out = torch.cat((x1, x2), dim=-1)
        out = self.fc(out)
        return out

model = Model()
model.eval()

x1 = torch.randn((2, 10, 32))
x2 = torch.randn((2, 10, 32))
x = (x1, x2)

torch.onnx.export(model,
                  x,
                  'model.onnx',
                  input_names=["input"],
                  output_names=["output"],
                  dynamic_axes={'input': {0: 'batch'}, 'output': {0: 'batch'}}
                  )
print("Done")

### Versions

Be like title!

**Comment by samdow** [comment]
Hi @singaln, we use github issues only for bug reports or feature requests. Please use the forum to ask questions: https://discuss.pytorch.org/

**Comment by BowenBao** [comment]
@singaln Considering inputs as `*args`, the correct way is to wrap an additional layer of tuple. 
After exporting to ONNX all inputs are flattened, so eventually it is treated as two tensor inputs in ONNX.

```python
torch.onnx.export(
    model,
    (x,),
    "model.onnx",
    input_names=["input", "input2"],
    output_names=["output"],
    dynamic_axes={
        "input": {0: "batch"},
        "input2": {0: "batch"},
        "output": {0: "batch"},
    },
)
```

**Comment by singaln** [comment]
> @singaln Considering inputs as `*args`, the correct way is to wrap an additional layer of tuple. After exporting to ONNX all inputs are flattened, so eventually it is treated as two tensor inputs in ONNX.
> 
> ```python
> torch.onnx.export(
>     model,
>     (x,),
>     "model.onnx",
>     input_names=["input", "input2"],
>     output_names=["output"],
>     dynamic_axes={
>         "input": {0: "batch"},
>         "input2": {0: "batch"},
>         "output": {0: "batch"},
>     },
> )
> ```

Thanks for your answer, I got it !
